<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Toolbox</class>
 <widget class="QDialog" name="Toolbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>402</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Toolbox</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>221</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Tools</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>29</x>
         <y>19</y>
         <width>161</width>
         <height>301</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="toolBtnReset">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toolBtnBlur">
          <property name="text">
           <string>Blur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toolBtnErode">
          <property name="text">
           <string>Erode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toolBtnDilate">
          <property name="text">
           <string>Dilate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toolBtnOpen">
          <property name="text">
           <string>Opening</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toolBtnClose">
          <property name="text">
           <string>Closing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toolBtnCannyFilter">
          <property name="text">
           <string>Canny Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toolBtnDetectEdge">
          <property name="text">
           <string>Detect Edges</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toolBtnDetectVertex">
          <property name="text">
           <string>Detect Vertices</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toolBtnDetectObject">
          <property name="text">
           <string>Detect Objects</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>Toolbox</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Toolbox</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
